naptha_sdk/agent.py

```diff
@@ -13,17 +13,17 @@ def __init__(self,
     ):
         self.orchestrator_run = orchestrator_run
         self.agent_index = agent_index
-        worker_node_url = self.orchestrator_run.orchestrator_deployment.agent_deployments[self.agent_index].worker_node_url
-        self.worker_node = Node(worker_node_url)
+        agent_node_url = self.orchestrator_run.agent_deployments[self.agent_index].agent_node_url
```

Comment:
As pointed out [here](https://github.com/NapthaAI/multiagent_chat/pull/1#discussion_r1898185187), could you please take a look at this again?